Readme.md

```diff
@@ -1 +1,57 @@
-Read Me.
+Project Details:
+
+Project Name: "Web Development Course Website"
+
+Project Description: "A website for beginers to learn HTML, CSS, Javascript, React And Node Js"
+
+Live Demo
+https://waseemqureshi2020.github.io/portfolio.github.io/
+
+Video Explainer:
+https://www.loom.com/share/b7f95d971daa4c88a8ba1dba7525d46d
+
+Built With:
+
+HTML
+CSS
+JAVA SCRIPT
+Linter
+
+Getting Started:
+
+First of all set up the .github folder, then inside it create a folder named as workflows, then inside that foloder create a yml file called linters.yml. Then take the corresponding github file for the HTML/CSS linter config,then add index.html and a style.css to style our main content.
+
+Prerequisites:
+node.js
+
+Setup:
+Through Github
+
+Install:
+Through Github
+
+Usage:
+Web Development Course
+
+Run tests:
+Through Github
+Linter test in local: npx stylelint "**/*.{css,scss}"
+
+Deployment
+Not deployed yet
+Authors
+👤 ** Waseem Qureshi **
+
+GitHub: @waseemqureshi2020
+Twitter: @WASEEMQ2020
+LinkedIn: Waseem Qureshi
+🤝 Contributing
+Thanks the microverse team their great contributions.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Original design idea by Cindy Shin in Behance.
+📝 License
+This project is MIT licensed.
```

Comment:
Great job @waseemqureshi2020 🤝

- [ ] I would suggest you use [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to insert links into your Readme file. This will make your project appear more professional.

Reply:
Thank you, I will keep that in mind for future projects. I have fixed it already.